explorer/src/pages/AccountDetailsPage.tsx

```diff
@@ -400,3 +463,114 @@ function getTabs(data?: ProgramData): Tab[] {
 
   return tabs;
 }
+
+function getCreatorDropdownItems(creators: Creator[] | null) {
+  const CreatorHeader = () => {
+    const shareTooltip =
+      "The percentage of the proceeds a creator receives when this NFT is sold.";
+
+    return (
+      <div
+        className={
+          "d-flex align-items-center dropdown-header creator-dropdown-entry"
+        }
+      >
+        <span className="dropdown-header text-monospace creator-dropdown-entry-address">
+          Creator Address
+        </span>
+        <div className="d-flex text-monospace">
+          <span className="text-monospace">Royalty</span>
+          <InfoTooltip bottom text={shareTooltip} />
+        </div>
+      </div>
+    );
+  };
+
+  const getVerifiedIcon = (isVerified: boolean) => {
+    const className = isVerified ? "fe fe-check" : "fe fe-alert-octagon";
+    return <i className={`ml-3 ${className}`}></i>;
+  };
+
+  const CreatorEntry = (creator: Creator) => {
+    return (
+      <div className={"d-flex align-items-center creator-dropdown-entry"}>
+        {getVerifiedIcon(creator.verified)}
```

Comment:
What does verified mean?

Reply:
If it's verified, it means that that account signed the metadata with a txn from that account. The command to do that is in the metaplex CLI tools and I believe it's built in to the storefront UI when you mint a new token. Since anyone can put any address in the creator array in the metadata JSON file, if the metadata is verified it means that the NFT was actually made by the accounts claiming to have made it in the creator array.